channels/rdpgfx/client/rdpgfx_main.c

```diff
@@ -1643,6 +1867,8 @@ static UINT rdpgfx_recv_pdu(RDPGFX_CHANNEL_CALLBACK* callback, wStream* s)
 				WLog_Print(gfx->log, WLOG_ERROR,
 				           "rdpgfx_recv_caps_confirm_pdu failed with error %" PRIu32 "!", error);
 
+			rdpgfx_send_cache_offer(gfx);
```

Comment:
return value shall be propagated ?

Reply:
I modified rdpgfx_send_cache_offer to return an error code that can be propagated like the rest